package.json

```
@@ -6,8 +6,9 @@
   "scripts": {
     "build": "bash scripts/build.sh",
     "storybook": "start-storybook -s .storybook/public -p 6006",
-    "docs:build": "compodoc src -p tsconfig.json",
-    "docs:server": "compodoc src -p tsconfig.json -s -w",
+    "docs:build": "compodoc -p tsconfig.json",
+	"docs:server": "compodoc -p tsconfig.json -s -w",
```

Tabs here, spaces elsewhere.
```suggestion
    "docs:server": "compodoc -p tsconfig.json -s -w",
```